homeassistant/components/weconnect/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""The WeConnect integration."""
+
+from dataclasses import dataclass
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .coordinator import WeConnectCoordinator
+
+PLATFORMS = [
+    Platform.SENSOR,
+]
+
+
+@dataclass
+class WeConnectData:
+    """Class to store WeConnect runtime data."""
+
+    coordinator: WeConnectCoordinator
+
+
+type WeConnectConfigEntry = ConfigEntry[WeConnectData]
```

Comment:
```suggestion
type WeConnectConfigEntry = ConfigEntry[WeConnectCoordinator]
```

No dataclass needed when there is only one object.